perma_web/perma/templates/user_management/settings-usage-plan.html

```diff
@@ -227,60 +227,58 @@ <h3 class="body-ch">Beginning {{ subscription.pending_change.effective | date:"F
-                <br>
-                ${{ package.amount }}
-                <br>
-                <br>
-                ${{ package.unit_cost | floatformat:2 }}/link
-              </button>
-              <input type="hidden" name="encrypted_data" value={{ package.encrypted_data }}>
-            </form>
-          {% endfor %}
-        </div>
-        {% if purchase_history.purchases %}
-          <hr>
-          <h3 class="body-ch">Purchase History</h3>
-          <dl class="dl-horizontal">
-            {% for purchase in purchase_history.purchases %}
-              <dt>
-                {{ purchase.date | date:"F n, Y" }}:
-              </dt>
-              <dd>
-                {{ purchase.reference_number }}
-              </dd>
-              <dd>
-                {{ purchase.link_quantity }} Links
-              </dd>
-            {% endfor %}
-          </dl>
-          <dl class="dl-horizontal">
+      <h2 class="body-ah">Get More Personal Links</h2>
+      <p class="page-dek">A one-time purchase, for people with or without subscriptions.</p>
+      <p class="fine-print">
+        Have a subscription? These extra links carry over. You can make them whenever you want, and you
+        can still make them if you cancel your subscription.
+      </p>
+      <div class="row">
+        {% for package in bonus_packages %}
+          <form class="purchase-form col-xs-12 col-md-3"
+                method="post"
+                action="{{ purchase_url }}">
+            <button>
+              <span>{{ package.link_quantity }} additional Links</span>
+              <br>
+              ${{ package.amount }}
+              <br>
+              <br>
+              ${{ package.unit_cost | floatformat:2 }}/link
+            </button>
+            <input type="hidden" name="encrypted_data" value={{ package.encrypted_data }}>
+          </form>
+        {% endfor %}
+      </div>
+      {% if purchase_history.purchases %}
+        <hr>
+        <h3 class="body-ch">Purchase History</h3>
+        <dl class="dl-horizontal">
+          {% for purchase in purchase_history.purchases %}
             <dt>
-              TOTAL
+              {{ purchase.date | date:"F n, Y" }}:
             </dt>
             <dd>
-              {{ purchase_history.total_links }} Links
+              {{ purchase.reference_number }}
             </dd>
-            <dt>
-              REMAINING
-            </dt>
             <dd>
-              {{ request.user.bonus_links }} Links
+              {{ purchase.link_quantity }} Links
             </dd>
-          </dl>
-        {% endif %}
+          {% endfor %}
+        </dl>
+        <dl class="dl-horizontal">
+          <dt>
+            TOTAL
+          </dt>
+          <dd>
+            {{ purchase_history.total_links }} Links
+          </dd>
+          <dt>
+            REMAINING
+          </dt>
+          <dd>
+            {{ request.user.bonus_links }} Links
+          </dd>
+        </dl>
```

Comment:
This diff is just removing `{% if ENABLE_BONUS_LINKS %}` and `{% endif %}` and outdenting: I didn't actually change anything else.